adr/configuration.py

```diff
@@ -66,20 +66,20 @@ def flatten(d, prefix=""):
     return sorted(result)
 
 
-class CustomCacheManager(CacheManager):
-    def __init__(self, adr_config):
+class ADRCacheManager(CacheManager):
```

Comment:
This is internal, I wouldn't change its name

Reply:
"Custom" is a very generic name, but ok